src/appshell/qml/Preferences/EditPreferencesPage.qml

```diff
@@ -22,16 +25,45 @@ PreferencesPage {
         width: parent.width
         spacing: root.sectionsSpacing
 
+        EffectBehaviorSection {
+            editPreferencesModel: editPreferencesModel
+
+            navigation.section: root.navigationSection
+            navigation.order: root.navigationOrderStart
+        }
+
+        SeparatorLine { }
+
+        AsymmetricStereoHeightsSection {
+            editPreferencesModel: editPreferencesModel
+
+            navigation.section: root.navigationSection
+            navigation.order: root.navigationOrderStart + 1
+        }
+
+        SeparatorLine { }
+
+        PastingBehaviorSection {
+            editPreferencesModel: editPreferencesModel
+
+            navigation.section: root.navigationSection
+            navigation.order: root.navigationOrderStart + 3 // skip one as there's navi for workspaces
```

Comment:
add navigationOrderEnd (alias to last navigation control) to AsymmetricStereoHeightsSection and reuse it here